Update/kage_001.txt

```diff
@@ -4962,7 +4962,7 @@ void main()
 //rそれは、暁自身から固く口止めされていたという建前上の理由もあったけど、…私が二人に課した、ちょっとした『試験』でもあったりする。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それは、暁自身から固く口止めされていたという建前上の理由もあったけど、…私が二人に課した、ちょっとした『試験』でもあったりする。",
-			NULL, "Akira himself was also very introverted... so I thought of their budding relationship as a little bit of a \"trial\" imposed on the two of them.", Line_Normal);
+			NULL, "Part of it was because Akira himself swore us to silence... but I also saw it as as a little bit of a \"trial\" imposed on the two of them.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Part of it was because Akira himself swore us to silence... but I also saw it as a little bit of a \"trial\" imposed on the two of them.", Line_Normal);
```